src/InputProcessor.java

```diff
@@ -0,0 +1,84 @@
+/*
+* https://docs.oracle.com/javase/tutorial/essential/environment/cmdLineArgs.html
+* https://stackoverflow.com/questions/513832/how-do-i-compare-strings-in-java
+* Bold text https://stackoverflow.com/a/29109958 - http://ascii-table.com/ansi-escape-sequences.php
+*/
+class InputProcessor {
+        void validateInput(String[] args) {
```

Comment:
We almost always use 4 spaces for indentation in Java, or alternatively use tabs.
Although personally I don't mind 8 spaces (because they force you to write methods with fewer degrees of indentation), this really is unconventional in Java.

Whether you choose 4 spaces or tabs, be sure to be consistent with all your code.

Reply:
I think this is ok now